kiosk/kiosk_browser/proxy.py

```diff
@@ -1,4 +1,56 @@
-"""Monitor proxy changes and automatically apply changes in Qt application"""
+"""Monitor proxy changes and automatically apply changes in Qt application.
+
+A ConnMan service, given from Dbus, has the following type:
+
+type service = [ path, service_config ]
+
+type service_config = dict {
```

Comment:
Would it make sense to actually use Python 3's typing features to write this as code instead of a comment?